File: Algorithms/1. Two Sum.java
public class Solution {
    public int[] twoSum(int[] nums, int target) {
        int res_l =-1;
        int res_r =-1;
        for(int i = 0;i < nums.length-1;i++){
            for(int k = i+1 ; k<nums.length; k++){
                if(nums[i]+nums[k] == target){
                    res_l=i;
                    res_r=k;
                    break;
                }
            }
            if(res_l != -1 && res_r != -1){
                break;
            }
        }
        return new int[]{res_l,res_r};
    }
}

File: Algorithms/1. Two Sum.rs
impl Solution {
    pub fn two_sum(nums: Vec<i32>, target: i32) -> Vec<i32> {
        let mut l = -1;
        let mut r = -1;
        for i in 0..nums.len() {
            for k in (i+1)..nums.len() {
                if nums[i] + nums[k] == target {
                    l = i as i32;
                    r = k as i32;
                    break;
                }
            }
            if l != -1 && r != -1 {
                break;
            }
        }
        vec![l, r]
    }
}
